claim.go

```diff
@@ -324,6 +335,18 @@ func (c *Claim) GetSchemaHash() SchemaHash {
 	return schemaHash
 }
 
+// GetSubjectPosition return a position where the subject is stored.
+func (c *Claim) GetSubjectPosition() (SubjectPosition, error) {
```

Comment:
Why do we need `SubjectPosition` type if we already have `Subject`?
I would propose just rename `getSubject` to `GetSubject` and that's it. Subject already a public type and all values of it are public too. I think we do not need another type. Return one of `Subject` constants would be fine.